.github/workflows/release-python-package.yml

```
@@ -28,11 +28,19 @@ jobs:
           node-version: '18.x'
           registry-url: 'https://registry.npmjs.org'
 
+      - name: Check file existence
+        id: check_files
+        uses: andstor/file-existence-action@v3
+        with:
+          files: "plugins/${{ inputs.package }}/src/deephaven/js/package.json"
```

I don't think `/deephaven` is on the path to the `package.json`
```suggestion
          files: "plugins/${{ inputs.package }}/src/js/package.json"
```